index.html

```diff
@@ -6,6 +6,392 @@
 	<link rel="stylesheet" type="text/css" href="index.css">
 </head>
 <body>
-	
+
+	<!-- intro -->
+	<header class="row">
+
+		<img src="assets/yellow-pill.svg" alt="yellow pill">
+		<img src="assets/more-pills.svg" alt="a blue pill, a purple pill, and a yellow pill">
+		
+		<span class="subTitle">A Viget Exploration</span>
+		<h1>
+			<img src="assets/title.svg" alt="Improving Medical Adherence">	
+		</h1>
+		
+		<img src="assets/pills.svg" alt="a blue pill and yellow pill">
+		<img src="assets/purple-pills.svg" alt="two purple pills">
+
+		<!-- yellow-pill.svg is top left -->
+		<!-- more-pills.svg is top right -->
+		<!-- pills.svg is bottom left -->
+		<!-- purple-pills.svg is bottom right -->
+
+		<section class="whiteBox">
+			<div>
+				<h2>Patients who don't take medication as prescribed are at risk of worsening health</h2>
+				<p>Just in the U.S., non-adherence has been attributed to over $100 billion dollars in avoidable healthcare costs.<span>1</span> In developed countries, among patients with chronic illness, approximately 50% do not take medications as prescribed.<span>2</span></p>
+				<p id="whiteBoxQuote">“Increasing the effectiveness of adherence interventions might have a greater impact on the health of the population than any improvement in specific medical treatment.”</p>
+				<p id="WHO">World Health Organization, 2005 <span>3</span></p>
```

Comment:
Please use classes for styling.  Ids should be reserved for grabbing DOM elements with JavaScript.